extensions/scarb-doc/tests/test.rs

```diff
@@ -0,0 +1,29 @@
+use scarb_test_support::command::Scarb;
+
+#[test]
+fn test_main() {
+    let output = Scarb::quick_snapbox()
+        .arg("doc")
+        .arg("--crate-path")
+        .arg("tests/hello_world.cairo")
```

Comment:
Consider enclosing the test data here, with use of https://docs.rs/assert_fs/latest/assert_fs/fixture/struct.TempDir.html